build/parameters.go

```diff
@@ -1,19 +1,86 @@
 package build
 
 import (
-	_ "embed"
+	"github.com/filecoin-project/go-state-types/abi"
+	actorstypes "github.com/filecoin-project/go-state-types/actors"
+
+	"github.com/filecoin-project/lotus/build/buildconstants"
+	proofparams "github.com/filecoin-project/lotus/build/proof-params"
+	"github.com/filecoin-project/lotus/chain/actors/policy"
 )
 
-//go:embed proof-params/parameters.json
-var params []byte
+var ParametersJSON = proofparams.ParametersJSON
+var SrsJSON = proofparams.SrsJSON
+
+// NOTE: DO NOT change this unless you REALLY know what you're doing. This is consensus critical.
+var BundleOverrides map[actorstypes.Version]string
```

Comment:
Noting that this has changed from being network-specific to global; but I think this is a positive change since there's nothing network-specific about it because it can only be changed with an env var.